packages/node_modules/@ciscospark/widget-space/src/actions.js

```diff
@@ -5,6 +5,7 @@ import messages from './messages';
 
 
 export const FETCH_SPACE_ERROR = `widget-space/FETCH_SPACE_ERROR`;
+export const STORE_TO_PERSON = `widget-space/STORE_TO_PERSON`;
```

Comment:
maybe at some future point we should reconsider "to person". i initially read this "point store to a person", not "store the 'to person'"